src/SFA.DAS.AdminService.Web/Handlers/Gateway/GetApplicationOverviewHandler.cs

```diff
@@ -71,13 +73,13 @@ public async Task<RoatpGatewayApplicationViewModel> Handle(GetApplicationOvervie
                     SequenceTitle = "Organisation checks",
                     Sections = new List<GatewaySection>
                     {
-                        new GatewaySection { SectionNumber = 1, PageId = "1-10", LinkTitle = "Legal name", HiddenText = "", Status = "" },
-                        new GatewaySection { SectionNumber = 2, PageId = "1-20", LinkTitle = "Trading name", HiddenText = "", Status = "" },
-                        new GatewaySection { SectionNumber = 3, PageId = "1-30", LinkTitle = "Organisation status", HiddenText = "", Status = "" },
-                        new GatewaySection { SectionNumber = 4, PageId = "1-40", LinkTitle = "Address", HiddenText = "", Status = "" },
-                        new GatewaySection { SectionNumber = 5, PageId = "1-50", LinkTitle = "ICO registration number", HiddenText = "", Status = "" },
-                        new GatewaySection { SectionNumber = 6, PageId = "1-60", LinkTitle = "Website address", HiddenText = "", Status = "" },
-                        new GatewaySection { SectionNumber = 7, PageId = "1-70", LinkTitle = "Organisation high risk", HiddenText = "", Status = "" }
+                        new GatewaySection { SectionNumber = 1, PageId = "1-10", UrlTag ="legal-name", LinkTitle = "Legal name", HiddenText = "", Status = "" },
```

Comment:
Are we removing these page ids?

Reply:
Out of interest, why do we need all of this config in the handler?

Reply:
We need a DevCuddle to figure out what to do about all the config (which is now in the overview).  I'm tackling the overview Monday so I'll think on it then, but I reckon it'll drop out during that redev